File: Arreglo.py
import random
 
arrayexample = [1,2,3,4,5]

for i in range(5):
    arrayexample[i]=int(random.uniform(100, 200))
    print(arrayexample[i-1])

    varInput=int(input("Ingrese el valor:"))
    print(varInput)

    for i in range(5):
        if arrayexample[i]>= varInput:
            print("el valor del arreglo es mayor")
        else:
            print("el valor del arreglo es menor"git init)  
        print(str(arrayexample[i])+">"+str(varInput))     

File: arreglo.js

let arrayExample = [1,2,3,4,5];
let inputVar = 178;

for (let i=0; i5;i++)  {
    arrayExample[i]=parseInt(Math.random()* (200 - 100)+100);
    console.log(arrayExample[i]);
}

for(let i=0;i<5;i++){
    if (arrayExample[i]>inputVar){
        console.log("El numero es mayor");
    }
    else{    
        console.log("El numero es menor");
    }    
}
